basic-javascript/project-1.spec.js

```diff
@@ -1,3 +1,140 @@
-const helpers = require('./project-1');
+const { multiplyByTen, subtractFive, areSameLength, areEqual, lessThanNinety, greaterThanFifty, add, subtract, divide, multiply, getRemainder, isEven, isOdd, square, cube, raiseToPower, roundNumber, roundUp, addExclamationPoint, combineNames, getGreeting, getRectangleArea, getTriangleArea, getCircleArea, getRectangularPrismVolume } = require('./project-1');
 
 // start testing!
+test('multiply by ten', () => {
+    const input = 10
+    const expected = 100
+    expect(multiplyByTen(input)).toEqual(expected)
```

Comment:
I wish You wrote some more tests for this one based on different conditions. What would happen if You passed different number? What would happen if You passed a string?